front/lib/resources/string_ids.ts

```diff
@@ -21,7 +21,8 @@ const REGION = 1; // US.
 const RESOURCES_PREFIX = {
   file: "fil",
   group: "grp",
-  vault: "vlt",
+  // TODO(2024-10-31 flav) Add new prefix for space.
```

Comment:
I think you can just remove the TODO no? Unlikely we'll ever get to this one no?

Reply:
I mean we could rewrite on the fly while we migrate maybe...

Reply:
We will, I was thinking about supporting two prefixes for one resource type. I'm all in, no more vaults 🙃.

Reply:
Until they rename it into something else :P